### PR TITLE
tidy up and added a gitignore

### PR DESCRIPTION
**Pull request by davidpayne1**
not sure about some of these files :\

### CODE REVIEW

File: src/Bede.AcceptanceTests.Common/packages.config [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Selenium.RC" version="2.45.0" targetFramework="net45" />
-  <package id="Selenium.WebDriver" version="2.45.0" targetFramework="net45" />
+  <package id="Selenium.Support" version="2.46.0" targetFramework="net45" />
+  <package id="Selenium.WebDriver" version="2.46.0" targetFramework="net45" />
```
Comment (nickmion) [inline src/Bede.AcceptanceTests.Common/packages.config:5]: Any reason for updating the version? I don't mind for the record, just curious?

Comment (davidpayne1) [inline src/Bede.AcceptanceTests.Common/packages.config:5]: is 'accident' a reason?

**Comment by nickmion** [comment]
You need to removed the obj folder files that where added

**Comment by DanielLackenbyBede** [comment]
Delete the package folder and commit once.
They'll be re-downloaded next time you build and it will remove them from source.

**Pull request merged**